src/features/header-flyout/HeaderFlyout.js

```diff
@@ -27,27 +29,27 @@ class HeaderFlyout extends React.Component<Props> {
 
     static defaultProps = {
         position: 'bottom-left',
+        isResponsive: false,
     };
 
     render() {
-        const { header, footer, flyoutButton, children, scrollRefFn, className, ...rest } = this.props;
+        const { header, footer, flyoutButton, children, isResponsive, scrollRefFn, className, ...rest } = this.props;
 
         return (
             <Flyout
-                closeOnClick={false}
-                offset={HeaderFlyout.panelOffset}
                 className={classNames('header-flyout', className)}
+                closeOnClick={false}
                 constrainToWindow
+                isResponsive={isResponsive}
+                offset={HeaderFlyout.panelOffset}
                 {...rest}
             >
                 {flyoutButton}
                 <Overlay className="header-flyout-overlay">
+                    <OverlayHeader>
+                        <h4 className="flyout-header-title">{header}</h4>
```

Comment:
I think this is a safeguard to prevent the CSS from applying if is-responsive-web is declared somewhere else. This is what the DejaBlue team did I believe